pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -950,7 +950,15 @@ const (
 	ProxyModeIPVS ProxyMode = "IPVS"
 )
 
-// KubeletConfig contains configuration settings for the kubelet.
+// ContainerLogSize determines container log rotation in kubelet
+type ContainerLogSize string
```

Comment:
Why we need a special type alias for this? Can the `DefaultContainerLogMaxSize` be var of type `resource.Quantity`?

Reply:
I can make it a string const

Reply:
done